Some bugfixes and registration notifications added

EyeOfProvidence7 opened the pull request.

EyeOfProvidence7 committed:
Some bugfixes and registration notifications added

JeanMeijer commented on Sources/WebApi/hippo_web/api.py:
```diff
@@ -41,6 +41,8 @@ def search_by_keywords(terms):
     for tweet in tweets:
         del tweet["raw"]
     
+    print("search for ", terms, " returned ", len(tweets), " result")
```
Debug statements, should be removed after usage.

JeanMeijer committed:
Update SearchBar.vue

JeanMeijer committed:
Added error notifications to search page.
Added consents to account page.
Bit of refactoring, made consent checked by default but allow opt-out.

JeanMeijer committed:
Removed debug statement.
